### PR TITLE
Fix bugs in stats library

### LINKED ISSUE #226: Stats: ViewData doesn't create new AggregationData
`ViewData` should have a mapping from `TagValue`s to `AggregationData`, rather than `Aggregation`. Calling `add()` on `Aggregation` will cause an error: https://github.com/census-instrumentation/opencensus-python/blob/master/opencensus/stats/view_data.py#L105-L108

### PR DESCRIPTION
**Pull request by liyanhui1228**
Fixes #226 #227

### CODE REVIEW

File: opencensus/stats/aggregation_data.py [hunk 1]
```diff
@@ -107,14 +107,21 @@ def __init__(self,
...
+                 counts_per_bucket=None,
+                 bounds=None):
         super(DistributionAggregationData, self).__init__(mean_data)
         self._mean_data = mean_data
         self._count_data = count_data
         self._min = min_
         self._max = max_
         self._sum_of_sqd_deviations = sum_of_sqd_deviations
+        if bounds is None:
+            bounds = [0]
```
[the remark this answers is not in the document]
Comment (liyanhui1228) [reply opencensus/stats/aggregation_data.py]: This part is mirror the behavior in here: https://github.com/census-instrumentation/opencensus-java/blob/8939d9fd639a8a7ff8b25e1a7e6c44d8b2e987db/impl_core/src/main/java/io/opencensus/implcore/stats/MutableAggregation.java#L251

[the remark this answers is not in the document]
Comment (songy23) [reply opencensus/stats/aggregation_data.py]: I see, you can leave the bounds as an empty list then.

File: examples/stats/helloworld/main.py [hunk 2]
```diff
@@ -61,6 +61,9 @@ def main():
     # Get aggregated stats and print it to console.
     view_data = view_manager.get_view(VIDEO_SIZE_VIEW_NAME)
     pprint(vars(view_data))   # TODO: print more meaningful info for view_data
```
Comment (songy23) [inline examples/stats/helloworld/main.py]: This TODO can be removed now.

Comment (liyanhui1228) [reply examples/stats/helloworld/main.py]: Removed.

**Pull request merged**